Mage/src/main/java/mage/abilities/condition/common/KickedCondition.java

```diff
@@ -5,24 +5,30 @@
 import mage.abilities.keyword.KickerAbility;
 import mage.game.Game;
 
-
 /**
  * Describes condition when spell was kicked.
  *
  * @author LevelX2
  */
 public enum KickedCondition implements Condition {
+    ONCE(1, ""),
+    TWICE(2, "twice");
+
+    private final int kickedCount;
+    private final String text;
 
-    instance;
+    KickedCondition(int kickedCount, String text) {
+        this.kickedCount = kickedCount;
+        this.text = text;
+    }
 
     @Override
     public boolean apply(Game game, Ability source) {
-        return KickerAbility.getSourceObjectKickedCount(game, source) > 0;
+        return KickerAbility.getSourceObjectKickedCount(game, source) >= kickedCount;
     }
 
     @Override
     public String toString() {
-        return "{this} was kicked";
+        return "{this} was kicked" + (text.isEmpty() ? "" : " ") + text;
```

Comment:
Could you change it to `(text.isEmpty() ? "" : " " + text)`. I think it would be more clear

Reply:
sure